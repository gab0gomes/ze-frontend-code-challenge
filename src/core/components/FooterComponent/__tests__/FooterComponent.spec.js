import { shallowMount, createLocalVue } from '@vue/test-utils'

import Footer from '../index'

const localVue = createLocalVue()

describe('FooterComponent', () => {
  it('render correctly', async () => {
    const wrapper = shallowMount(Footer, {
      localVue
    })

    expect(wrapper.element).toMatchSnapshot()

    wrapper.destroy()
  })
})
