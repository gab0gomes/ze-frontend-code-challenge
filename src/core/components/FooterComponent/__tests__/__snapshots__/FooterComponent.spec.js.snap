// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent render correctly 1`] = `
<footer>
  <div
    class="container"
  >
    <div
      class="logo-container"
    >
      <logo-stub
        class="logo"
      />
       
      <span>
        Frontend Challenge
      </span>
    </div>
     
    <div
      class="company-address"
    >
      <p>
        ZX COMMERCE COMERCIO DE BEBIDAS LTDA - CNPJ: 37.657.197/0001-46
      </p>
       
      <p>
        Rua Fradique Coutinho, 1632 - CEP: 05.416-002
      </p>
       
      <p>
        Pinheiros - SÃ£o Paulo/SP
      </p>
    </div>
  </div>
</footer>
`;
